### PR TITLE
Issues-2-havrylenko

### PR DESCRIPTION
**Pull request by YuliiaHavrylenko**
### **Add WeeklyReportFile model**
Were created a new model for storing weekly reports from services(Uklon, Uber, Bolt) in the database.
Have been added function:

- transfer_reports_to_db (query)
- check_full_data (checking the number of days in a report)
- convert_file_name (simplifies getting the date from the filename)
- save_weekly_reports_to_db (the main function that checks all conditions by calling other functions and saves files to the database)

### **Test add test file and settings for Pytest**
To give the test access to the database, need to execute the command to the database:
ALTER USER universal_auto_dev CREATEDB;
To start testing need to execute the command: pytest

### CODE REVIEW

File: app/migrations/0005_weeklyreportfile.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        migrations.CreateModel(
+            name='WeeklyReportFile',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('organization_name', models.CharField(max_length=20)),
+                ('report_file_name', models.CharField(max_length=255, unique=True)),
+                ('report_from', models.CharField(max_length=10)),
+                ('report_to', models.CharField(max_length=10)),
+                ('file', models.TextField()),
+                ('saved_at', models.DateTimeField(auto_now_add=True)),
```
Comment (SergeyKutsko) [inline app/migrations/0005_weeklyreportfile.py:22]: @YuliiaHavrylenko rename to standart `created_at`

File: app/models.py [hunk 2]
```diff
@@ -127,28 +145,143 @@ def save_uber_report_to_db(file_name):
...
+        self.organization_name = company_name
+        self.report_file_name = report_name
+        self.report_from = from_date
+        self.report_to = until_date
+        self.file = (header + rows)
+        self.save()
+
+
+# Calculates the number of days in the report
+def check_full_data(start, end, file_name):
```
Comment (SergeyKutsko) [inline app/models.py:191]: @YuliiaHavrylenko please move to class

File: app/models.py [hunk 3]
```diff
@@ -127,28 +145,143 @@ def save_uber_report_to_db(file_name):
...
+    difference = end - start
+    if difference.days == days:
+        return True
+    else:
+        print(f"{file_name} include {difference.days} days of the week")
+        return False
+
+
+# Help separate the date from file name
+def convert_file_name(split_symbol, name_list):
```
Comment (SergeyKutsko) [inline app/models.py:204]: @YuliiaHavrylenko please move to class

File: app/models.py [hunk 4]
```diff
@@ -127,28 +145,143 @@ def save_uber_report_to_db(file_name):
...
+    converted_list = []
+    for string in name_list:
+        string = string.split(split_symbol)
+        for part in string:
+            converted_list.append(part)
+    return converted_list
+
+
+def save_weekly_reports_to_db():
```
Comment (SergeyKutsko) [inline app/models.py:213]: @YuliiaHavrylenko please move to class

File: app/models.py [hunk 5]
```diff
@@ -127,28 +145,143 @@ def save_uber_report_to_db(file_name):
...
+        self.report_file_name = report_name
+        self.report_from = from_date
+        self.report_to = until_date
+        self.file = (header + rows)
+        self.save()
+
+
+# Calculates the number of days in the report
+def check_full_data(start, end, file_name):
+    days = 7
```
Comment (SergeyKutsko) [inline app/models.py:192]: @YuliiaHavrylenko why we need that variable?

File: app/models.py [hunk 6]
```diff
@@ -1,4 +1,13 @@
+import csv
+from datetime import datetime
+import glob
+import os
 from django.db import models
+import csv
+import os
+import glob
```
Comment (SergeyKutsko) [inline app/models.py:8]: @YuliiaHavrylenko remove extra line

File: app/models.py [hunk 7]
```diff
@@ -1,4 +1,13 @@
+import csv
+from datetime import datetime
+import glob
+import os
 from django.db import models
+import csv
+import os
```
Comment (SergeyKutsko) [inline app/models.py:7]: @YuliiaHavrylenko remove extra line

File: app/models.py [hunk 8]
```diff
@@ -1,4 +1,13 @@
+import csv
+from datetime import datetime
+import glob
+import os
 from django.db import models
+import csv
```
Comment (SergeyKutsko) [inline app/models.py:6]: @YuliiaHavrylenko remove extra line

**Comment by YuliiaHavrylenko** [comment]
@SergeyKutsko as far as I have checked, all the requested changes have been made.

**Review by SergeyKutsko — APPROVED** [review]
@YuliiaHavrylenko congrats pick your time for Interview
https://calendly.com/universal_auto_internship/30min

**Pull request merged**